packages/components/src/tools-panel/stories/typography-panel.js

```diff
@@ -0,0 +1,217 @@
+/**
+ * WordPress dependencies
+ */
+import {
+	__experimentalFontAppearanceControl as FontAppearanceControl,
+	__experimentalFontFamilyControl as FontFamilyControl,
+	__experimentalLetterSpacingControl as LetterSpacingControl,
+	LineHeightControl,
+} from '@wordpress/block-editor';
```

Comment:
What is the reasoning behind picking these controls from `@wordpress/block-editor`, but picking `FontSizePicker` from `@wordpress/components` ?

In general, for the purpose of this Storybook example, should we use components from `@wordpress/block-editor` or from `@wordpress/components`?

Reply:
I just imported the components used in the [existing `TypographyPanel`](https://github.com/WordPress/gutenberg/blob/cdc5c152554773c340dbcc386066d1b96a5b0143/packages/edit-site/src/components/global-styles/typography-panel.js) for the most part.

I don't think it really matters which things we use here (existing block-editor components or wp-component primitives) at least in this PR, as they are only placeholders right now. We'll be evaluating each piece on a case-by-case basis, and replace when necessary. As a starting point, I'm inclined toward having the "real world" block-editor components as a reference so we don't make assumptions based on static mockups only. (e.g. This has already surfaced a concern with the Letter Spacing mockup, which doesn't take units into account.)